ping/rust/src/bin/testplan_0500.rs

```diff
@@ -2,30 +2,47 @@ use anyhow::Result;
 use async_trait::async_trait;
 use futures::StreamExt;
 use libp2pv0500::swarm::{keep_alive, NetworkBehaviour, SwarmEvent};
+use libp2pv0500::{tokio_development_transport,webrtc};
+use libp2pv0500::core::multiaddr::*;
+use libp2pv0500::core::muxing::*;
 use libp2pv0500::*;
-use std::collections::HashSet;
-use std::time::Duration;
-use testplan::{run_ping, PingSwarm};
+use rand::thread_rng;
+use std::{
+    collections::HashSet,
+    time::Duration,
+};
+use log::info;
+use testplan::*;
 
 #[async_std::main]
 async fn main() -> Result<()> {
     let local_key = identity::Keypair::generate_ed25519();
     let local_peer_id = PeerId::from(local_key.public());
-
-    let swarm = OrphanRuleWorkaround(Swarm::with_async_std_executor(
-        development_transport(local_key).await?,
+    let client = testground::client::Client::new_and_init().await.unwrap();
```

Comment:
ah, I see. It's because of the return type of `new_and_init` no worries John.

>Is there a particular reason why unwrapping in main is problematic? The program is going to exit with a non-zero return code either way. If I did an expect it'll have a message and stack trace.

it's mostly consistency, as we have the `-> Result<()>` kinda implies we should be returning a Result